tasks/audit-cat1.yml

```diff
@@ -121,9 +121,9 @@
       - RHEL-07-040800
       - always
 
-- name: "HIGH | RHEL-07-010340 | AUDIT | RHEL-07-040390 | PATCH | The SSH daemon must be configured to only use the SSHv2 protocol"
+- name: "HIGH | RHEL-07-010340 | AUDIT | Verify the operating system requires users to supply a password for privilege escalation"
   stat:
-      command: grep -i nopasswd /etc/sudoers /etc/sudoers.d/*
+      command: grep -i NOPASSWD /etc/sudoers /etc/sudoers.d/*
```

Comment:
Use the find module to address this, not command: grep. 

See RHEL-07-020100 for an example.